viral/offline_sequence_detection.py

```
@@ -0,0 +1,564 @@
+        trials_train, trials_test = train_test_split(
+            trials, train_size=train_size, random_state=42
+        )
+    else:
+        # otherwise just keep all imaged trials
+        trials_train = trials
+
+    # do the place cell template only on the training data!
+    session.trials = trials_train
+
+    cache_file = Path(
+        f"{SERVER_PATH}/viral_caches/sequence_detection/{session.mouse_name}_{session.date}_{f"online" if bayesian_config.online else "offline"}_{f"en_bloc" if bayesian_config.en_bloc else "per_event"}_place_cells.npz"
```

Ruff (0.14.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)